README.md

```diff
@@ -1,54 +1,266 @@
+
+[![gradient_black_24dp 1](https://user-images.githubusercontent.com/48277920/146035893-6611f42d-f7aa-40c9-8889-fa812d6ce878.png#gh-light-mode-only)](https://github.com/bouvet/BouvetDevelopmentKit/wiki#gh-light-mode-only)[Optimized shader](https://github.com/bouvet/BouvetDevelopmentKit/wiki#gh-light-mode-only)
 
-## Supported version
+[![palette_black_24dp 2](https://user-images.githubusercontent.com/48277920/146035891-77c566f4-64de-4cdd-b914-c8fc5dde0989.png#gh-light-mode-only)](https://github.com/bouvet/BouvetDevelopmentKit/wiki#gh-light-mode-only)[Color profile](https://github.com/bouvet/BouvetDevelopmentKit/wiki#gh-light-mode-only)
 
-BDK has only been tested with the following versions of Unity and Visual studio so far. There are plans to move to Unity 2021 once it enters LTS, but unitll then support will only be experimental.
+<br/>
+<br/>
 
-- Visual studio 2019
-- Unity 2020.4 LTS
+# Samples
 
-## Installation via Git in UPM
+<table width="100%">
+	<tr>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#floating-menu">
+				<img align="center" width="450" src="https://raw.githubusercontent.com/wiki/bouvet/BouvetDevelopmentKit/images/BDK_01_FloatingMenu.gif" />
+			</a>
+		</td>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#hand-menu">
+				<img align="center" width="450" src="https://raw.githubusercontent.com/wiki/bouvet/BouvetDevelopmentKit/images/BDK_01_HandMenu_1.gif" />
+			</a>
+		</td>
+	</tr>
+	<tr>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#floating-menu">
+				Floating Menu
+			</a>			
+			<br/>
+			The floating menu is a type of panel that can hold any content or menu.
+		</td>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#hand-menu">
+				Hand Menu
+			</a>
+			<br/>
+			The hand menu is another common and simple menu.
+		</td>
+	</tr>
+	<tr>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#clock-menu">
+				<img align="center" width="450" src="https://raw.githubusercontent.com/wiki/bouvet/BouvetDevelopmentKit/images/BDK_04_ClockMenu_01.gif" />
+			</a>
+		</td>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#drape-menu">
+				<img align="center" width="450" src="https://raw.githubusercontent.com/wiki/bouvet/BouvetDevelopmentKit/images/BDK_03_DrapeMenu_02.gif" />
+			</a>
+		</td>
+	</tr>
+	<tr>
+		<td width="50%">
+			<a href="https://github.com/bouvet/BouvetDevelopmentKit/wiki/Essentials#clock-menu">
+				Clock Menu
+			</a>			
+			<br/>
+			The clock menu prefab is an example of a menu that appears where a watch would normally be.
```

Comment:
The clock menu ~~prefab is an example of a menu that~~ appears where a watch would normally be.